src/components/PageButtons/PageButtons.test.jsx

```diff
@@ -0,0 +1,29 @@
+import renderer from 'react-test-renderer';
+import PageButtons from '.';
+
+const createInput = function createInput(prevPage, nextPage) {
+  return {
+    prevPage,
+    nextPage,
+    selectedTrack: 't',
+    selectedCohort: 'c',
+    getPrevNextGrades() {},
+  };
+};
+
+describe('PageButtons', () => {
+  it('renders empty links as disabled', () => {
```

Comment:
Maybe we should break this test case into multiple test cases? Also, you are repeating statements in these tests a lot, is there a way to refactor?